### PR TITLE
Export the EVBlock on the entry type

### PR DESCRIPTION
**Pull request by john-floren-gravwell**
It was causing a problem for encoding to be unexported.

### CODE REVIEW

File: chancacher/chancacher_test.go [hunk 1]
```diff
@@ -697,6 +703,136 @@ func TestCacheMaxSize(t *testing.T) {
 	}
 }
 
+// TestCacheEntries verifies that we can write entries, with EVs
+// attached, and read them back out.
+func TestCacheEntries(t *testing.T) {
+	gob.Register(&entry.Entry{})
```
Comment (kris-watts-gravwell) [inline chancacher/chancacher_test.go:709]: if you have to register them in this test we probably need to register them in the main chancacher package.

**Review by kris-watts-gravwell — COMMENTED** [review]
LGTM, questions about the gob.Register call.  Seems like we probably need to do that in the entry package?

File: chancacher/chancacher_test.go [hunk 2]
```diff
@@ -16,6 +16,12 @@ import (
 	"strings"
 	"testing"
 	"time"
+
+	"fmt"
+	"net"
+
+	"github.com/gravwell/gravwell/v3/ingest/entry"
```
[the remark this answers is not in the document]
Comment (john-floren-gravwell) [reply chancacher/chancacher_test.go:23]: I'm trying out the emacs Go modes and it seems to be lazier about how it groups imports.

**Review by kris-watts-gravwell — APPROVED** [review]
LGTM

**Pull request merged**